src/test/java/org/sagebionetworks/bridge/spring/controllers/ParticipantReportControllerTest.java

```diff
@@ -476,6 +481,46 @@ public void nonAdminCannotDeleteParticipantIndex() {
         controller.deleteParticipantReportIndex(REPORT_ID);
     }
     
+    @Test(expectedExceptions = EntityNotFoundException.class, 
+            expectedExceptionsMessageRegExp=".*Account not found.*")
+    public void getParticipantReportAccountNotFound() {
+        doReturn(session).when(controller).getAuthenticatedSession(RESEARCHER);
+        reset(mockAccountService);
+        controller.getParticipantReport(USER_ID, REPORT_ID, null, null);
+    }
+    
+    @Test(expectedExceptions = EntityNotFoundException.class, 
+            expectedExceptionsMessageRegExp=".*Account not found.*")
+    public void getParticipantReportForWorkerAccountNotFound() {
+        doReturn(session).when(controller).getAuthenticatedSession(WORKER);
+        reset(mockAccountService);
+        controller.getParticipantReportForWorker(TEST_STUDY_IDENTIFIER, USER_ID, REPORT_ID, null, null);
+    }
```

Comment:
Unit tests for the v4 versions of this API?

Reply:
Added